main.ts

```diff
@@ -62,6 +62,16 @@ class Translator {
     this.emit(')');
   }
 
+  visitExternalModuleReferenceExpr(expr: ts.Expression) {
+    // TODO: what if this isn't a string literal?
+    var moduleName = <ts.StringLiteralExpression>expr;
+    if (moduleName.text.lastIndexOf("./", 0) !== 0) {
```

Comment:
Yes.
startsWith isn't available, we're not currently running node with --harmony. Probably should do, since we are using target: ts.ScriptTarget.ES6
For now, using the regex approach.